packages/shepherd/build.sh

```diff
@@ -0,0 +1,51 @@
+TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/shepherd/
+TERMUX_PKG_DESCRIPTION='The Shepherd—extensible service manager'
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.0.5
+TERMUX_PKG_SRCURL=https://codeberg.org/shepherd/shepherd/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=f09f0dab1fd0cd11988e5069be96eb3601182e94db7fed8cf328f99de506306c
+TERMUX_PKG_DEPENDS="guile, guile-fibers"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--with-gzip=$TERMUX_PREFIX/bin/gzip --with-zstd=$TERMUX_PREFIX/bin/zstd"
+TERMUX_PKG_RM_AFTER_INSTALL="
+bin/reboot
+bin/halt
+bin/shutdown
+share/man/man8/halt.8
+share/man/man8/reboot.8
+"
+
+termux_step_host_build() {
+	FIBERS_SCRIP="$TERMUX_SCRIPTDIR/packages/guile-fibers/build.sh"
+	FIBERS_VERSION=$(grep '^TERMUX_PKG_VERSION=' "$FIBERS_SCRIP" | awk -F'=' '{print $2}')
+	FIBERS_URL=$(grep '^TERMUX_PKG_SRCURL=' "$FIBERS_SCRIP" | awk -F'=' '{print $2}' | sed "s/\${TERMUX_PKG_VERSION}/$FIBERS_VERSION/")
+	FIBERS_SHA256=$(grep '^TERMUX_PKG_SHA256=' "$FIBERS_SCRIP" | awk -F'=' '{print $2}')
```

Comment:
We usually prefer sourcing the build script when we need a variable from it.
https://github.com/termux/termux-packages/blob/1bad6eac3e697f1a8d109f9c2f656ebacd151f48/packages/ncurses/build.sh#L28-L32
https://github.com/termux/termux-packages/blob/1bad6eac3e697f1a8d109f9c2f656ebacd151f48/packages/gnupg/build.sh#L26-L29